.github/ISSUE_TEMPLATE/update-hive.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Update Hive
+about: >-
+  This template contains instructions specific to updating this product and/or
+  container image(s).
+title: >-
+  chore(hive): Update container images ahead of Stackable Release XX.(X)X
+labels: []
+projects: ['stackabletech/10']
+assignees: ''
+---
+
+Part of #xxx.
+
+> [!TIP]
+> Please add the `scheduled-for/20XX-XX` label.
+
+```[tasklist]
+### Update tasks
+- [ ] Update `versions.py` to reflect the agreed upon versions in the spreadsheet (including the removal of old versions).
+- [ ] Upload new version (see `hive/upload_new_hive_version.sh`).
+- [ ] Update `versions.py` to the latest supported version of JVM (base and devel).
+- [ ] Update other dependencies if applicable (eg: jmx_exporter, aws_java_sdk_bundle, etc).
+- [ ] Check other operators (getting_started / kuttl) for usage of the versions. Add the PR(s) to the list below.
+- [ ] Update the version in demos. Add the PR(s) to the list below.
+```
+
+```[tasklist]
+### Related Pull Requests
+- [ ] _Link to the docker-images PR (product update)_
+- [ ] _Link to [hdfs-utils](https://github.com/stackabletech/hdfs-utils/) PR (if applicable)_
```

Comment:
Don't we need this rather in the hdfs template?

Reply:
I think this was copy-pasta. Will check...

Reply:
Fixed in 39dc3ae2b9d9c9953cfe84581e0f2650223db1b3